lib/logflare/utils.ex

```diff
@@ -80,6 +80,29 @@ defmodule Logflare.Utils do
   def stringify(v) when is_binary(v), do: v
   def stringify(v), do: inspect(v)
 
+  @doc """
+  Stringifies a term.
+
+  ### Example
+    iex> stringify(:my_atom)
+    "my_atom"
+    iex> stringify(1.1)
+    "1.1"
+    iex> stringify(122)
+    "122"
+    iex> stringify("something")
+    "something"
+    iex> stringify(%{})
+    "%{}"
+    iex> stringify([])
+    "[]"
+  """
+  def stringify(v) when is_atom(v), do: Atom.to_string(v)
```

Comment:
Probably overkill, but worth noting that there is a guard I added recently in `Logflare.Utils.Guards` called `is_atom_value` which will exclude atoms such as `nil` or boolean values which can sometimes leak through to general utility functions like this 🤷‍♂️ 

Up to @Ziinc if that level of concern is needed here though.